classes/class3/homework/submit_casts/app.js

```diff
@@ -0,0 +1,36 @@
+var express = require('express');
+var app = express();
+
+var path = require('path')
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var index = require('./routes/index.js')
+var exphbs = require('express-handlebars');
+var PORT = process.env.PORT || 3000;
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.engine('handlebars', exphbs({defaultLayout: 'main'}));
+app.set('view engine', 'handlebars');
+
+app.get('/',function(req,res){
+	res.render("home",{'paths':['/casts','/casts/new','/casts/bymember/Adam Archer','/casts/delete/old']});
```

Comment:
It's bad practice to have a space in a url parameter. (`'/casts/bymember/Adam Archer'`)
It is getting automatically URL encoded by the browser here, so it's working as you would expect (also because of the error you have below), but that won't be true of all user agents. Stick to single word route parameters in the future.